### PR TITLE
Further comments from Tom Petch and Mohamed Boucadair

### PR DESCRIPTION
**Pull request by michael-scharf**
This addresses ...

- The remaining comments from Tom Petch (as far as not addressed already), see https://mailarchive.ietf.org/arch/msg/tcpm/JQ3JjuVmKx40R8fH9Q2PKDpJEEo/
- The editorial comments from Mohamed Boucadair, see https://mailarchive.ietf.org/arch/msg/tcpm/nNqEcdpw1nZ7UcsslR7AVCuhjn4/

### CODE REVIEW

File: draft/draft-ietf-tcpm-yang-tcp.xml [hunk 1]
```diff
@@ -176,7 +190,7 @@
 
             <t>Connection parameters: Many implementations have means to
             influence the behavior of each TCP connection, e.g., on the
-            programming interface used by applications. A typical example are
+            programming interface used by applications. Typical examples are
```
Comment (mjethanandani) [inline draft/draft-ietf-tcpm-yang-tcp.xml:193]: This is one example only. Did you mean to add more examples here, and why you make the sentence plural?

File: draft/draft-ietf-tcpm-yang-tcp.xml [hunk 2]
```diff
@@ -259,7 +280,7 @@
       </section>
 
       <section title="Tree Diagram">
-        <t>This section provides a abridged tree diagram for the YANG module
+        <t>This section provides a tree diagram for the YANG module
```
Comment (mjethanandani) [inline draft/draft-ietf-tcpm-yang-tcp.xml:283]: Truth be told, it is an abridged tree diagram. We decided to include the full tree diagram in the appendix.

Comment (michael-scharf) [reply draft/draft-ietf-tcpm-yang-tcp.xml]: OK, I have just reverted to the original text. This was a comment from Med.

File: draft/draft-ietf-tcpm-yang-tcp.xml (hunk 1 above)
Comment (michael-scharf) [reply draft/draft-ietf-tcpm-yang-tcp.xml:193]: This was a review comment from Med. But as there are many different socket options (see next line), plural somehow makes sense to me. The document currently just mentions one such socket option (TCP_NODELAY), but there are more.

I am not a native speaker. Do I miss something?

**Review by mjethanandani — APPROVED** [review]

**Pull request merged**